README.md

```diff
@@ -205,69 +205,56 @@ docker run --rm --name pg1 -e PGSO_INSTANCE_NAME=pg1 -e PGSO_REGION=eu-north-1 \
   pgspotops/pg-spot-operator:latest
 ```
 
-## Cleanup of all operator created objects
-
-After some work/testing one can clean up all operator created cloud resources via `PGSO_TEARDOWN_REGION` or only a
-single instance via the `PGSO_TEARDOWN` flag.
+# Security
 
-```
-docker run --rm --name pg1 -e PGSO_TEARDOWN_REGION=y -e PGSO_REGION=eu-north-1 \
-  -e PGSO_AWS_ACCESS_KEY_ID="$(grep -m1 aws_access_key_id ~/.aws/credentials | sed 's/aws_access_key_id = //')" \
-  -e PGSO_AWS_SECRET_ACCESS_KEY="$(grep -m1 aws_secret_access_key ~/.aws/credentials | sed 's/aws_secret_access_key = //')" \
-  pgspotops/pg-spot-operator:latest
-```
+By default, security is not super trimmed down, as the experience is geared towards more casual or temporary workloads.
+Everything is tunable though.
 
-or by running a helper script from the "scripts" folders (by default just lists the resources, need to add a parameter):
+**ATTENTION** For all public Spot Operator managed instances it is very much **recommended to create a new special
```

Comment:
what does public in `public Spot Operator managed instances` mean?

Reply:
Added that its about a publice IP